src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -196,13 +196,13 @@ class InterpolatedVolatilitySurface(BaseModel):
     All fitted vols, the z-axis in the mesh.
     """
 
-    input_params: Dict[AnyStr, object]
+    input_params: Dict[str, object]
```

Comment:
AnyStr is producing dict with bytes as its key and thus failing pydantic serialization.
@kdowney-cloudwallcapital any concern if we change to basic str?

Reply:
No concerns, it's the more proper representation for abstract types, but if it's breaking Pydantic, change it.